cordova_push.mli

```diff
@@ -328,6 +328,16 @@ val clear_all_notifications :
   (unit -> unit)  ->
   unit
 [@@js.call]
+
+(* iOS & Android only *)
+val clear_notification :
+  t               ->
+  (unit -> unit)  ->
+  (unit -> unit)  ->
+  int             ->
```

Comment:
I would rather say:

```suggestion
  int             ->
  (unit -> unit)  ->
  (unit -> unit)  ->
```

According to the [documentation of the Cordova plugin](https://github.com/havesource/cordova-plugin-push/blob/2.0.0/docs/API.md#pushclearnotificationid-successhandler-errorhandler---ios--android-only).

Reply:
This is what I did initially and realised it does not work. 
The documentation indeed says that we should call clearNotification this way:
push.clearNotification(id, successHandler, errorHandler) 

But if you look at this link https://github.com/havesource/cordova-plugin-push/blob/master/src/js/push.js and check the clearNotification definition, you will see:
clearNotification (successCallback = () => {}, errorCallback = () => {}, id) 
Therefore the right order for the parameters is the order of the parameters section in the documentation.

Reply:
Indeed! Good catch!